### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by joaobranco17** [openPMD/openPMD-viewer#142]
Currently the pip install is failing with the following error 
```
pip install openPMD-viewer
Collecting openPMD-viewer
  Using cached openPMD-viewer-0.5.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-2_8yy6dj/openPMD-viewer/setup.py", line 47, in <module>
        "opmd_viewer/openpmd_timeseries/cython_function.pyx"),
      File "/data/home/branco77/python/data_analysis/lib/python3.4/site-packages/Cython/Build/Dependencies.py", line 818, in cythonize
        aliases=aliases)
      File "/data/home/branco77/python/data_analysis/lib/python3.4/site-packages/Cython/Build/Dependencies.py", line 704, in create_extension_list
        for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
      File "/data/home/branco77/python/data_analysis/lib/python3.4/site-packages/Cython/Build/Dependencies.py", line 108, in nonempty
        raise ValueError(error_msg)
    ValueError: 'opmd_viewer/openpmd_timeseries/cython_function.pyx' doesn't match any files
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-2_8yy6dj/openPMD-viewer/
```

**Comment by RemiLehe** [comment]
Hi Joao,

Thanks for reporting the issue. I am currently working on it, but it might take me a little while to figure out how to fix the problem.

In the meantime, there are two temporary solutions that you can use on your side:
- If you do not need the new particle tracking functionality, then you can roll back to version 0.4.0 of `openPMD-viewer`:
```
pip install openPMD-viewer==0.4.0
```
which should install without any problem.
- Another option is to use `conda` to install `openPMD-viewer`:
```
conda install -c rlehe openpmd_viewer
```
The `conda` install works even with the new version that has particle tracking.

**Comment by PrometheusPi** [comment]
@RemiLehe The problem with pip was solved by installing scipy via pip beforehand. Then we could install openPMD-viewer without any problems.

**Comment by RemiLehe** [comment]
OK, thanks for letting me know. That's hard to explain, but I'm glad it worked anyway.

In the meantime, I identified other problems with the install, and fixed them in PR #143 . There should be less installation problems once this is fixed.

**Comment by PrometheusPi** [comment]
@RemiLehe Thanks for fixing the problem so fast. 
Perhaps scipy installed Cython - or our attempt to install openPMD by hand required Cython and we thus installed it. I do not remember and I have no access to @joaobranco17 system right now to check it.
